### ISSUE TITLE
Support group-level service-accounts

### ISSUE THREAD

**Issue by sven-hergenhahn-dmtech** [python-gitlab/python-gitlab#2812]
Since 16.1, the gitlab-api supports the creation of group-level service-accounts and group-level service-account tokens (They are not the same as group access tokens!)
So far they can not yet be created via the Web UI.

It would be great to have this feature added to python-gitlab.

The gitlab API documentation is here: https://docs.gitlab.com/ee/api/groups.html#service-accounts

**Comment by nejch** [comment]
Service accounts were added in #2851 - we can track adding the tokens here though.

**Comment by svenXY** [comment]
I don't think that service-account-tokens have been added meanwhile, so this is still something I'd like to see in python-gitlab

**Comment by svenXY** [comment]
I don't think that service-account-tokens have been added meanwhile, so this is still something I'd like to see in python-gitlab

**Comment by svenXY** [comment]
I don't think that service-account-tokens have been added meanwhile, so this is still something I'd like to see in python-gitlab

**Comment by syphernl** [comment]
There is [an open PR](https://github.com/python-gitlab/python-gitlab/pull/3109) that adds this functionality, but it has been stalled for a long time now..

**Comment by sven-hergenhahn-dmtech** [comment]
I don't think that service-account-tokens have been added meanwhile, so this is still something I'd like to see in python-gitlab

**Comment by syphernl** [comment]
There has been some activity in the PR 2 months ago but since then nothing happened: https://github.com/python-gitlab/python-gitlab/pull/3109

While this issue may be "stale", this functionality is still something multiple users would like to see.

**Comment by syphernl** [comment]
There is still a PR waiting for review..

https://github.com/python-gitlab/python-gitlab/pull/3361
